RecoLocalTracker/SiPixelClusterizer/plugins/gpuCalibPixel.h

```diff
@@ -23,9 +23,9 @@ namespace gpuCalibPixel {
   constexpr float VCaltoElectronOffset_L1 = -670;  // L1:   -670 +- 220
   constexpr int VCalChargeThreshold = 100;
   //for phase2
-  constexpr float ElectronPerADCGain = 600;
+  constexpr float ElectronPerADCGain = 1500;
   constexpr int8_t Phase2ReadoutMode = 3;
-  constexpr uint16_t Phase2DigiBaseline = 1500;
+  constexpr uint16_t Phase2DigiBaseline = 1000;
```

Comment:
These are the only physics-related changes I've seen so far (I've skipped `Geometry/CommonTopologies/interface/SimplePixelTopology.h` for the time being).

Shouldn't they be mentioned in the description of the PR ?